network-api/networkapi/buyersguide/translation.py

```diff
@@ -0,0 +1,64 @@
+from .pagemodels.products.base import Product
+from .pagemodels.products.general import GeneralProduct
+from .pagemodels.products.software import SoftwareProduct
+from .pagemodels.product_category import BuyersGuideProductCategory
+
+from modeltranslation.translator import TranslationOptions
+from modeltranslation.decorators import register
+
+from networkapi.buyersguide.pagemodels.privacy import ProductPrivacyPolicyLink
+
+
+@register(Product)
+class ProductTR(TranslationOptions):
+    fields = (
+        'name',
+        'company',
+        'blurb',
+        'worst_case',
+        'signup_requirement_explanation',
+        'personal_data_collected',
+        'biometric_data_collected',
+        'social_data_collected',
+        'how_does_it_use_data_collected',
+        'how_can_you_control_your_data',
+        'track_record_details',
+        'offline_use_description',
+        'uses_encryption_helptext',
+        'security_updates_helptext',
+        'strong_password_helptext',
+        'manage_vulnerabilities_helptext',
+        'privacy_policy_helptext',
+        'ai_helptext',
+    )
+
+
+@register(GeneralProduct)
+class GeneralProductTR(TranslationOptions):
+    fields = (
+        'child_rules_helptext',
```

Comment:
Are these confirmed as needing localization?

Reply:
If it’s not used anymore, it can be dropped.
Also note that without that line, General products are not localized. There must be something else to do to make it work, though

Reply:
@kalob we'll probably need to verify this against https://docs.google.com/spreadsheets/d/1cOOzLhkoICFIPTUmxQnrH10HZz1qq5eHF51qgj1p8QM/edit#gid=0 (and, I guess the actual code on pni-v4-2020 now =D)

Reply:
I've compared this to the fields in the main pni-v4-2020 branch and made sure all the text-based fields (TextField, CharField, etc) have the right translation fields, and made sure all the translation fields exist.

I'm going to resolve this for now, but if there's more or I misinterpreted this task let's open another review comment.